pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -123,6 +123,8 @@ public static AggregationFunction getAggregationFunction(FunctionContext functio
             return new DistinctCountAggregationFunction(firstArgument);
           case DISTINCTCOUNTBITMAP:
             return new DistinctCountBitmapAggregationFunction(firstArgument);
+          case PARTITIONEDDISTINCTCOUNT:
```

Comment:
we need to start using _ separators. I think the parser now handles both automatically

Reply:
Discussed offline. We should still keep no underscore within the aggregation function type, and parser will remove the underscore from the input function name to match the function type.